site/src/pages/community/articles.mdx

```diff
@@ -20,9 +20,24 @@ Send a pull request by editing [this page](https://github.com/line/armeria/edit/
   date="20220414"
   type="slides"
   description="
-    Introduces how to migrate your Spring Servlet application to Armeria. This talk includes a performance
-    highlight during the LINE Channel's migration which enables a large payload download API in the web
-    application server without any concern about thread exhaustion or OutOfMemoryException.
+    Learn about the 3 steps for migrating your traditional Spring MVC application to Armeria.
+    This talk also gives a performance highlight on the LINE Channel's migration that enabled
+    the team to build a large payload download API without any concern about thread exhaustion
+    or OutOfMemoryException.
+  "
+/>
+
+<Article
+  title="Building an Authentication Microservice at LINE"
+  url="https://drive.google.com/file/d/1M2W75eYsd8Vk2LCh2BYpbRRWGZBXw6fP/view"
+  author="Dohan Kim"
+  date="20220414"
+  type="slides"
+  description="
+    This talk unveils Doorkeeper, the authentication microservice that is responsible for
```

Comment:
This seems like a cool project to open source 😆

Reply:
Indeed. There was a similar question at the webinar, but it looked like they didn't consider that option in the beginning, i.e. tightly coupled with the internals.

Reply:
Oh~ though it is beginning phase, I think we can consider various way 😄